public/app/plugins/panel/news/NewsPanel.tsx

```diff
@@ -0,0 +1,128 @@
+// Libraries
+import React, { PureComponent } from 'react';
+import RssParser from 'rss-parser';
+import { css } from 'emotion';
+
+// Utils & Services
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory, CustomScrollbar, styleMixins } from '@grafana/ui';
+import config from 'app/core/config';
+import { feedToDataFrame } from './utils';
+import { sanitize } from 'app/core/utils/text';
+
+// Types
+import { PanelProps, DataFrameView, dateTime } from '@grafana/data';
+import { RssFeed, NewsOptions, NewsItem } from './types';
+
+interface Props extends PanelProps<NewsOptions> {}
+
+interface State {
+  news?: DataFrameView<NewsItem>;
+  isError?: boolean;
+}
+
+const CORS_PROXY = 'https://cors-anywhere.herokuapp.com/';
```

Comment:
Ha, never heard of this, amazing!

Reply:
I'm a bit nervous about adding it automatically though -- after we have lenient CORS headers set on our feed, I think we should just recommend it in the UI rather than adding it automatically

Reply:
Yeah, that's a good thinking